ocs_ci/ocs/utils.py

```diff
@@ -746,7 +746,32 @@ def run_must_gather(log_dir_path, image, command=None):
         )
 
 
-def collect_ocs_logs(dir_name, ocp=True, ocs=True):
+def collect_noobaa_db_dump(log_dir_path):
+    """
+    Collect the Noobaa DB dump
+
+    Args:
+        log_dir_path (str): directory for dumped Noobaa DB
+
+    """
+    from ocs_ci.ocs.resources.pod import get_pods_having_label, download_file_from_pod, Pod
```

Comment:
Any reason why we are importing here and not at the top of the file?

Reply:
because of a cicular import loop. See what happens when I move the import line to the top of the file:

```Traceback (most recent call last):
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 575, in import_plugin
    __import__(importspec)
  File "/home/Elad/git/ocs-ci/ocs_ci/framework/pytest_customization/ocscilib.py", line 26, in <module>
    from ocs_ci.ocs.utils import collect_ocs_logs, collect_prometheus_metrics
  File "/home/Elad/git/ocs-ci/ocs_ci/ocs/utils.py", line 31, in <module>
    from ocs_ci.ocs.resources.pod import get_pods_having_label, download_file_from_pod, Pod
  File "/home/Elad/git/ocs-ci/ocs_ci/ocs/resources/pod.py", line 17, in <module>
    from tests import helpers
ImportError: cannot import name 'helpers' from 'tests' (/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/tests/__init__.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/Elad/git/ocs-ci/venv/bin/run-ci", line 11, in <module>
    load_entry_point('ocs-ci', 'console_scripts', 'run-ci')()
  File "/home/Elad/git/ocs-ci/ocs_ci/framework/main.py", line 119, in main
    return pytest.main(arguments)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 73, in main
    config = _prepareconfig(args, plugins)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 214, in _prepareconfig
    config = get_config(args, plugins)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 186, in get_config
    pluginmanager.consider_preparse(args)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 509, in consider_preparse
    self.consider_pluginarg(parg)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 534, in consider_pluginarg
    self.import_plugin(arg, consider_entry_points=True)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 583, in import_plugin
    raise new_exc.with_traceback(tb)
  File "/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/_pytest/config/__init__.py", line 575, in import_plugin
    __import__(importspec)
  File "/home/Elad/git/ocs-ci/ocs_ci/framework/pytest_customization/ocscilib.py", line 26, in <module>
    from ocs_ci.ocs.utils import collect_ocs_logs, collect_prometheus_metrics
  File "/home/Elad/git/ocs-ci/ocs_ci/ocs/utils.py", line 31, in <module>
    from ocs_ci.ocs.resources.pod import get_pods_having_label, download_file_from_pod, Pod
  File "/home/Elad/git/ocs-ci/ocs_ci/ocs/resources/pod.py", line 17, in <module>
    from tests import helpers
ImportError: Error importing plugin "ocs_ci.framework.pytest_customization.ocscilib": cannot import name 'helpers' from 'tests' (/home/Elad/git/ocs-ci/venv/lib64/python3.7/site-packages/tests/__init__.py)```

Reply:
I think the issue here is that we are importing a test helper from `resources.pod`. If we have something in `tests.helpers` that needs to be used in something that isn't a test case then it should live in our `utility` modules instead. We don't need to fix this in this PR but I'm fairly certain this is what is causing the circular import.

Reply:
+1